includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -167,6 +170,8 @@ public function register() {
 				$this->contribute_with_google_block->register();
 				$this->subscribe_with_google_block->register();
 			}
+
+			add_action( 'admin_enqueue_scripts', $this->get_method_proxy( 'enqueue_assets_for_non_sitekit_user_on_block_editor_page' ), 40 );
```

Comment:
Why are we not using the `enqueue_block_editor_assets` hook here instead? That way, we shouldn't need the `is_block_editor_page` method at all. Please correct me if I'm wrong, thanks!

Reply:
Thanks @nfmohit, that's a good question! I was actually taking a cue from how the `Asset_Bar` [sets up](https://github.com/google/site-kit-wp/blob/419f2d26c7e44de662eba814f63abc65f900e02f/includes/Core/Admin_Bar/Admin_Bar.php#L108) the enqueued scripts and styles, but only because it didn't occur to me to try using `enqueue_block_editor_assets` instead here.

If that works, which it seems on the face of it that it should, we can indeed get rid of `is_block_editor_page()`, which would be great. As you've [suggested](https://github.com/google/site-kit-wp/issues/10400#issuecomment-2715403126) we can figure this out during execution.